### ISSUE TITLE
Allow `output` and `assert` in the same command

### ISSUE THREAD

**Issue by sswan** [sswan/thresh#7]
Allow something like

```
thresh foo.txt cat u1=col2 u2=col1 output gen.csv assert "np.linalg.norm(u1-u2) < 1.0e-6"
thresh foo.txt cat u1=col2 u2=col1 assert "np.linalg.norm(u1-u2) < 1.0e-6" output gen.csv
```

**Comment by eric-c-cyr** [comment]
Can we also have output to standard out?

**Comment by sswan** [comment]
@eric-c-cyr Good point. Printing to stdout would be `print` and not `output`. Would you want to have all these options at once (output, print, assert)?

```
thresh foo.txt cat u1=col2 u2=col1 print .csv output gen.csv assert "np.linalg.norm(u1-u2) < 1.0e-6"
```

**Comment by sswan** [comment]
However, `print` and `assert` both want to print to the screen. I've been trying to make it so that only one "task" is printing at a time. This could be overcome by making sure that `assert` is always printing to stderr and `print` is always to stdout.

@eric-c-cyr What do you think?

**Comment by eric-c-cyr** [comment]
For my current use case, print would be sufficient. Though since I"m being greedy...yes all at once
